src/plugins/data_source/config.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { schema, TypeOf } from '@osd/config-schema';
+
+export const configSchema = schema.object({
+  enabled: schema.boolean({ defaultValue: false }),
+  wrappingKeyName: schema.string({
+    minLength: 1,
+    maxLength: 100,
+    defaultValue: 'wrappingKeyName',
```

Comment:
what use this for default value?

Reply:
> why use this for default value?

the default value is provided for simplify the configuration, but we can also remove it

Reply:
ok with default value, but could we come up better value

Reply:
> meanwhile, WRAPPING_KEY_SIZE is 32, will this default value for with 32 byte key size?

I'm not sure I understand the last part, but I believe the question is whether the value here means setting the default size. It's config validation, which means that if the size of the config file is less than 32, it will throw an error. And it must be 32, as defined by the encryption algorithm for key wrapping. The generated data key is not 32 bytes in length.